packages/react-i18n/src/i18n.ts

```diff
@@ -44,6 +44,29 @@ const DECIMAL_NOT_SUPPORTED = 'N/A';
 const PERIOD = '.';
 const DECIMAL_VALUE_FOR_CURRENCIES_WITHOUT_DECIMALS = '00';
 
+// Specifies the number of fractional digits for those currencies that don't use two digits.
+const CURRENCY_DIGITS = {
```

Comment:
There's already a utility imported for this, [currencyDecimalPlaces](https://github.com/Shopify/quilt/blob/master/packages/react-i18n/src/constants/index.ts#L179-L182).

Reply:
Good point--thank you!  Adjusting as suggested.